rfcs/text/0052/gen_ai.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- name: gen_ai
+  fields:
+    - name: system_instructions
+      type: flattened
+      description: The system message or instructions provided to the GenAI model separately from the chat history.
+      example: TODO
```

Comment:
update all examples

Reply:
I pushed the changes, @trisch-me with all examples!